data/p3/p3.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+
+import numpy
+import time
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+from sklearn.svm import SVC
+from sklearn.datasets import load_svmlight_file
+
+from utils import save_plot
+
+
+def load_data(path):
+    with open(path, 'rb') as f:
+        return load_svmlight_file(f, n_features=784)
```

Comment:
this load returns a scipy.sparse.csr_matrix object, you don't want to train svm with objects
see https://docs.scipy.org/doc/scipy-0.19.0/reference/generated/scipy.sparse.csr_matrix.todense.html